data/en/test_radio_mandatory_with_mandatory_other.json

```diff
@@ -0,0 +1,59 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.1",
```

Comment:
All test schemas should use `data_version` 0.0.2 so we don't have to provide q_code properties in the answers.